### DISCUSSION TITLE
error in ui-lovelace.yaml

### DISCUSSION THREAD

**Discussion by bachoo786** [matt8707/hass-config#174] [Q&A]
Hi there

I have followed the instructions and restarted my HA. I get this error:

`in "/config/ui-lovelace.yaml", line 141, column 15: Unable to read file /config/popup/vardagsrum_tv.yaml`

Can you help please? thanks.

**Comment by angr72** [comment]
Do you have the file /config/popup/vardagsrum_tv.yaml stored on that path?
